web/shared/src/poll_data.ts

```diff
@@ -55,17 +139,12 @@ export class PollData {
                         return;
                     }
 
-                    if (!Number.isInteger(idx) || idx < 0 || idx > MAX_IDX) {
+                    if (idx < 0 || idx > MAX_IDX) {
                         this.report_error_function("poll widget: bad type for inbound option idx");
```

Comment:
`idx` won't be non-Number now so I think the error message can be tailored to something like "idx out of boud"

Reply:
Yeah agreed, Changed it!